### ISSUE TITLE
Change style for code tag

### ISSUE THREAD

**Issue by mazulo** [dunderlabs/dunderlabs.github.io#10]

**Comment by nltncsr** [comment]
Hm, I understand. Something lighter would be better.

**Comment by mazulo** [comment]
Yeah, something like the tags codes in this post http://blog.dunderlabs.com/classes-e-objetos-i.html

**Comment by nltncsr** [comment]
Would it be like in [this commit](https://github.com/dunderlabs/pure/tree/843e599a0ff309bf97d54d122bc2d40bf4e55086), lighter? Or did I understand it wrong and it should be darker? :smiley:

**Comment by mazulo** [comment]
When I clicked in that link, I was redirect to GitHub page of DunderLabs. Would it the real reference at this [link](https://github.com/dunderlabs/pure/commit/843e599a0ff309bf97d54d122bc2d40bf4e55086)? If so I think it should be a little darker

**Comment by nltncsr** [comment]
Odd. [Here I successfully enter](blog.dunderlabs.com/tag/translations.html) a tag page. What tags are not working?!
